spec/cypress/support/factorybot.js

```diff
@@ -9,19 +9,96 @@ class FactoryBot {
    * @param args The arguments to pass to FactoryBot.create(), e.g.
    * factory name, traits, and attributes. Pass them in as separated
    * string arguments. Attributes should be passed as an object.
-   * @returns The FactoryBot.create() response
-   *
-   * @example
+   * You are also able to call instance methods on the created record later.
+   * @returns The FactoryBot.create() response.
+   * @examples
    * FactoryBot.create("factory_name", "with_trait", { another_attribute: ".pdf"})
+   * FactoryBot.create("factory_name").then(res => {res.call.any_rails_method(42)})
```

Comment:
Even though a little out of scope, this mightbe a good time to also add an example where an array of is passed, e.g. something like

```
FactoryBot.create("tutorial", { lecture_id: this.lecture.id, tutor_ids: [this.tutor1.id, this.tutor2.id] })
```

or similar.